lib/src/over_react_test/custom_matchers.dart

```diff
@@ -398,8 +400,8 @@ class _LoggingFunctionMatcher extends CustomMatcher {
 /// caught error.
 ///
 /// Related: [logsToConsole], [hasNoLogs]
-Matcher hasLog(dynamic expected, {ConsoleConfiguration consoleConfig}) =>
-    _LoggingFunctionMatcher(anyElement(contains(expected)), config: consoleConfig);
+Matcher hasLog(dynamic expected, {ConsoleConfiguration consoleConfig, bool ddcOnly = false}) =>
```

Comment:
Sorry, I only mentioned it briefly in my last comment, but could we update this name (`ddcOnly`) as well, here and in other places?

Reply:
Addressed in https://github.com/Workiva/over_react_test/pull/104/commits/d5b3fb455a79881e7c724f37a0fb76dbac9cb09dv